### ISSUE TITLE
optional belongsTo relation set to null throws "couldn't find model with id"

### ISSUE THREAD

**Issue by remkoboschker** [samselikoff/ember-cli-mirage#645]
Hi,

I have a one-2-many relationship from `package` to `item`, but not all items belong to a package. When I define fixtures for packages and items and leave the `packageId` empty or set it to null for the items without a package. The setter in `addMethodsToModelClass` in `belongs-to.js` raises an error because it cannot find a fixture with the id null.

````
  assert(
          !id || schema[camelize(association.modelName)].find(id),
          `Couldn\'t find ${association.modelName} with id = ' + id`
        );
```` 
Is an optional relation not supported?

**Comment by samselikoff** [comment]
This should only happen if you did something like

```
item.packageId = '2'
```

but there was no `package` with that id. Null foreign keys should work.

**Comment by remkoboschker** [comment]
Thank you for the quick reply. I modified the logging to `Couldn\'t find ${association.modelName} with id = ${id}` and found out that in fact there was an id there. (The model could not be found because I forgot the give the fixture file a plural name). I make a little pull request to improve the log.

**Comment by remkoboschker** [comment]
It was already fixed by f92f4f42e76433a8491db11b5ae7a59586479f1b

**Comment by samselikoff** [comment]
Glad you figured it out, and thanks for reporting this!
